### ISSUE TITLE
Facebook/Instagram bought Giphy. Giphy is built into Wire. We don't want Facebook data-mongers in Wire.

### ISSUE THREAD

**Issue by tokariu** [wireapp/wire#347]
Here is the official news:
https://about.fb.com/news/2020/05/welcome-giphy/

Please discuss why or why not this is a serious issue for Wire.

**Comment by CharlesCage** [comment]
Came to express similar concern, though I suspect Wire has a plan to remove/replace it.

**Comment by cardassian-tailor** [comment]
Post this in another section on wires GitHub also. I agree 100% 

The tricky part is - you need to team up with a company who provides a similar service but has a dedication and commitment to privacy. I don’t know of one. The only other alternative is to create a new product in that space. If you do that you have a new problem - the model for gif sharing services is primarily a freemium model based around ads and selling personal data. You would have to reinvent that. 

I’d say go with the later. Team up with other players in this space. Signal. Proton. Wire. Get these people on the same wave length and make a product to fill this space that also respects privacy.

**Comment by jamestomasino** [comment]
A wire for iOS issue from a few years ago linked to [this post](https://signal.org/blog/giphy-experiment/) from Signal about their experiments in building a secure interface to giphy. Perhaps there is a path to be followed or new developments in the area.

**Comment by tokariu** [comment]
> A wire for iOS issue from a few years ago linked to [this post](https://signal.org/blog/giphy-experiment/) from Signal about their experiments in building a secure interface to giphy. Perhaps there is a path to be followed or new developments in the area.

indeed, I also remember the signal-solution. That might be a great solution handling the giphy situation.
I guess this is more realistic than privacy oriented messengers teaming up to create a privacy oriented giphy-clone. but who knows...

what's the wire-team thinking about this?

**Comment by vincegre** [comment]
> what's the wire-team thinking about this?

Probably absolutely nothing considering how they completely gave up on their client softwares since years :(
Myself I would see absolutely no objection removing that useless feature for me !

**Comment by kesenwang** [comment]
That sounds really concerning. Anything to do with Facebook I hope Wire distances itself from.

**Comment by vincegre** [comment]
> That sounds really concerning. Anything to do with Facebook I hope Wire distances itself from.

Don't hope anything from Wire, it's a long time they have completely given up on getting a client properly coded and that works fine (same bugs since ages in app....) and they don't care about feedback from customers....

**Comment by kesenwang** [comment]
> > That sounds really concerning. Anything to do with Facebook I hope Wire distances itself from.
> 
> Don't hope anything from Wire, it's a long time they have completely given up on getting a client properly coded and that works fine (same bugs since ages in app....) and they don't care about feedback from customers....

Yeah, I've been feeling that disconnect for the past year too now. It's pretty much like the iOS app has been abandoned and they went full steam ahead for enterprise only. I used to use it for all messaging, don't really use it at all anymore.

**Comment by vincegre** [comment]
> Yeah, I've been feeling that disconnect for the past year too now.
Past year ? you're really kind with them, they have never given a sh*t about their users and improve their crappy app and it's like that since at least 4 years !!
